docs/core/development/policy/release-process.rst

```diff
@@ -102,48 +102,47 @@ Prepare for a release
    - ``git checkout -b release-$RELEASE-4290``
 
 
-How to do a pre-release
------------------------
+How to do a release candidate
+-----------------------------
 
 1. Check ​buildbot to make sure all supported platforms are green (wait for pending builds if necessary).
 2. If a previously supported platform does not currently have a buildbot, move from supported platforms to "expected to work" in ``INSTALL.rst``.
 3. In your Git repo, fetch and check out the new release branch.
-4. Run ``python -m incremental.update Twisted --newversion $RELEASErc1``
+4. Run ``python -m incremental.update Twisted --rc``
```

Comment:
Might be nice if this was a simple script in `admin/` so if this command changes, it's obvious that you need to fix the script, and this doc don't have to change.